apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -1535,6 +1536,29 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
                         !showManualInviteTable
                             ? (
                                 <>
+                                    {
+                                        !userConfig?.enableBulkImportSecondaryUserStore
+                                        && (
+                                            <Grid.Row columns={ 1 } className="mb-0 pb-0">
+                                                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
```

Comment:
If the higher breakpoint values are the same, no need to define them explicitly. Smaller breakpoint value will be inherited. Please check other places as well.
```suggestion
                                                <Grid.Column mobile={ 16 }>
```

Reply:
Improved with e5131107b219625042ca86f56ec14449e7752cd5.